### ISSUE TITLE
README.md and Wiki driver instructions conflict

### ISSUE THREAD

**Issue by idontusenumbers** [conductor-framework/conductor#37]
The readme says to place drivers in resources folder, the wiki says to put them in project root.

When the driver is missing, a nondescript error comes up with a wiki link, but the exception is swallowed and the 'file not found' information is lost.

**Comment by ddavison** [comment]
yes, the original implementation was to use the root directory as the driver location.  it's since been changed to use the resources directory.  an MR would be greatly appreciated! 👍

**Comment by idontusenumbers** [comment]
I first started using the resources folder and it didnt work at all; but it did work with the project root.
